Configuration/TCA/tx_academicpersons_domain_model_organisational_unit.php

```diff
@@ -121,26 +116,6 @@
                 'type' => 'text',
             ],
         ],
-        'valid_from' => [
```

Comment:
Property, getter and setter is not removed from corresponding `Fgtclb\AcademicPersons\Domain\Model\OrganisationalUnit` Domain
Model.

Which could be breaking. I would emphasize this using

```
[!!!][TASK]
```

prefix for commit and pull-request.

Reply:
valid_from is kept.